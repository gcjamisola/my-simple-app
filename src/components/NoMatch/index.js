import React, { Component } from 'react';

class NoMatch extends Component {
	render() {
		return (
			<h2>NoMatch</h2>
		);
	}
}

export default NoMatch;