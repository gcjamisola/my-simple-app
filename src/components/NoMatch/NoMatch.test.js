import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import NoMatch from './index';

describe('NoMatch component', () => {
	it('should render the component', () => {
		const component = shallow(<NoMatch />);
		const tree = toJson(component);
		expect(tree).toMatchSnapshot();
	});
});