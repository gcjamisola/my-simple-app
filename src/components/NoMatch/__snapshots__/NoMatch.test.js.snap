// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch component should render the component 1`] = `
<h2>
  NoMatch
</h2>
`;
